lib/config.js

```diff
@@ -16,35 +16,49 @@ const Config = {
   schema: {
     autocomplete: {
       title: 'Enable Autocomplete',
+      includeTitle: false,
```

Comment:
Interesting, on my machine I don't see a difference between `includeTitle: false,` and the default setting.

If it fixes a issue on your machine, let's go with it 👍